packages/model-registry/upstream/frontend/src/odh/components/DeployModalExtension.tsx

```diff
@@ -36,46 +31,37 @@ const DeployModalExtension: React.FC<DeployModalExtensionProps> = ({ mv, render
     [extensionsLoaded, extensions],
   );
 
-  const handleSubmit = React.useCallback(() => {
-    setOpenModal(false);
-    // Redirect to deployments tab of the model version page after successful deployment
-    const modelVersionId = mv.id;
-    const registeredModelId = mv.registeredModelId;
-    const modelRegistryName = preferredModelRegistry?.name;
-    
-    navigate(modelVersionDeploymentsUrl(modelVersionId, registeredModelId, modelRegistryName));
```

Comment:
+1 to @katieperry4 's comment
> I did notice that once you deploy it takes you back to the model registry main page, it would be nice if it took you to the registered models deploy page
You can use this `modelVersionDeploymentsUrl` from the old code to get the URL you need.